src/app/components/header-nav/header-nav.component.css

```diff
@@ -0,0 +1,27 @@
+.menu {
+    align-items: center;
+    background-color:#ffebca ;
```

Comment:
better: background-color: #FFEBCA;
note the space, and all the hex values on Uppercase